pkg/manager/manager.go

```diff
@@ -158,6 +169,51 @@ func StartManager() {
 		os.Exit(1)
 	}
 
+	// Make sure TLS cert watcher is already set up.
+	if tlsWatcherReady == nil {
+		setupLog.Error(err, "tls watcher not ready")
```

Comment:
due to recent supportablity feedback , suggestion for more information in the error message. If customer runs into this issue, what are the action item to resolve the issue.